src/main/java/com/mpatric/mp3agic/AbstractID3v2Tag.java

```diff
@@ -958,16 +958,16 @@ public void setUrl(String url) {
 	}
 
 	@Override
-	public ArrayList<ID3v2ChapterFrameData> getChapters() {
+	public List<ID3v2ChapterFrameData> getChapters() {
 		if (obseleteFormat) {
-			return null;
+			return Collections.emptyList();
```

Comment:
Changing from returning null to returning an empty collection changes the SDK's interface, which might affect applications that are using it. I'd prefer not to make a breaking change unless it's absolutely neccessary.

Reply:
You have a valid point. I agree.
- We should probably add a new test that checks that this method implementation returns a null-value. I presumed the tests would fail if breaking changes were made. They all passed.
- Alternatively we could create a new implementation that returns an empty list and mark the old method as  `@Deprecated`

Reply:
@mpatric Went up with implementing a new interface that extends the old one, please see 202b0b0. This solution does not break existing code.

Reply:
> We should probably add a new test that checks that this method implementation returns a null-value. I presumed the tests would fail if breaking changes were made. They all passed.

Done. Please refer to bd77447